lms/static/certificates/sass/_components.scss

```diff
@@ -569,3 +572,89 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
```

Comment:
Please use a depth value from our Pattern Library's z-depths Sass map (https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Futilities%2F_variables.scss#L242). The function you would call is here - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Futilities%2F_functions.scss#L57.